app/models/user.js

```diff
@@ -0,0 +1,20 @@
+import Model, { attr } from '@ember-data/model';
+
+export default class UserModel extends Model {
+  @attr('string', { defaultValue: 'User' }) first_name;
```

Comment:
Why are we setting first_name default value to `User`

Reply:
I think we can remove the default first_name